packages/wagmi/src/client.ts

```diff
@@ -191,7 +191,7 @@ export class Web3Modal extends Web3ModalScaffold {
       onChange: connectors => this.syncConnectors(connectors)
     })
     watchAccount(this.wagmiConfig, {
-      onChange: accountData => this.syncAccount({ ...accountData, config: wagmiConfig })
+      onChange: accountData => this.syncAccount({ ...accountData })
```

Comment:
Is this correct?

Reply:
hm I'm 90% sure it is since it's not being used in the code but @glitch-txs do you happen to remember why we had this ?